msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -14,10 +14,10 @@ keywords: events bot channel message reaction conversation
 
 When building your conversational bots for Microsoft Teams, you can work with conversation events. Teams sends notifications to your bot for conversation events that happen in scopes where your bot is active. You can capture these events in your code and take the following actions:
 
-* Trigger a welcome message when your bot is added to a team.
-* Trigger a welcome message when a new team member is added or removed.
-* Trigger a notification when a channel is created, renamed, or deleted.
-* Trigger a notification when a bot message is liked by a user.
+* Trigger a welcome message when a new team member is added or removed
```

Comment:
Ojasvi will help word another bullet point that captures the addition of select channel info